app/uk/gov/hmrc/digitalservicestax/views/financial_details.scala.html

```diff
@@ -0,0 +1,70 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(lineItems: List[FinancialTransaction], messages: UniformMessages[Html])
+
+@formatter = @{java.time.format.DateTimeFormatter.ofPattern("d MMM YYYY")}
+@cash = @{java.text.NumberFormat.getInstance.format(_: BigDecimal)}
+<div class="govuk-grid-column-full">
```

Comment:
Can you make this with two-thirds width, as that is what it's like on the prototype.